MetaMask/metamask-mobile
# feat: add consensys zkevm (Linea) support

=======

**Description**
Add support for ConsenSys zkEVM network (Linea).

**Screenshots/Recordings**
<img width="291" alt="Screenshot 2023-03-15 at 2 00 32 PM" src="https://user-images.githubusercontent.com/85494462/225399835-8d3cf280-bf05-4fe4-8992-804d6d5b271d.png">

https://user-images.githubusercontent.com/85494462/227060203-6dbee5d6-09ca-4adc-9726-7e3164fc21e9.mp4


**Acceptance Criteria**
## Manual Testing Steps

Step 1:
Go to this website: https://bridge.goerli.zkevm.consensys.net/send?token=ETH and send some ETH (minimum 0.1) from Goerli to Consensys zkEVM (Linea) .
<img width="1908" alt="Screenshot 2023-02-23 at 12 04 31 PM" src="https://user-images.githubusercontent.com/85494462/220888987-def69536-19de-433f-bf11-6170c25eab1e.png">

Wait for transaction to be finished.

Step 2:
Open the metamask mobile app and select then new ConsenSys zkEVM (Linea) test network in the dropdown.
<img width="291" alt="Screenshot 2023-03-15 at 2 00 32 PM" src="https://user-images.githubusercontent.com/85494462/225399835-8d3cf280-bf05-4fe4-8992-804d6d5b271d.png">

Step 3:
Go to this website: https://bridge.goerli.zkevm.consensys.net/send?token=ETH and send some ETH (minimum 0.1) from  Consensys zkEVM (Linea) to Goerli.
<img width="1914" alt="Screenshot 2023-02-23 at 12 07 30 PM" src="https://user-images.githubusercontent.com/85494462/220889588-0561bce0-3e23-43fa-acea-3396084d989c.png">

Wait for transaction to be finished.

Final step:
Check if all transactions succeed on Metamask.
- What Dapps does it support?

**Related Tickets**
Core PR (https://github.com/MetaMask/core/pull/1126) to follow 

**Checklist**
